src/beta_inc.jl

```diff
@@ -0,0 +1,876 @@
+using Base.Math: @horner
+using Base.MPFR: ROUNDING_MODE
+const exparg_n = log(nextfloat(floatmin(Float64)))
+const exparg_p =  log(prevfloat(floatmax(Float64)))
+
+#COMPUTE log(gamma(b)/gamma(a+b)) when b >= 8
+"""
+    loggammadiv(a,b)
+
+Computes ``log(\\Gamma(b)/\\Gamma(a+b))`` when b >= 8
+"""
+function loggammadiv(a::Float64, b::Float64)
+    if a > b
+        h = b/a
+        c = 1.0/(1.0 + h)
+        x = h/(1.0 + h)
+        d = a + (b - 0.5)
+        x² = x*x
+        s₃ = 1.0 + (x + x²)
+        s₅ = 1.0 + (x + x²*s₃)
+        s₇ = 1.0 + (x + x²*s₅)
+        s₉ = 1.0 + (x + x²*s₇)
+        s₁₁ = 1.0 + (x + x²*s₉)
+    else
+        h = a/b
+        c = h/(1.0 + h)
+        x = 1.0/(1.0 + h)
+        d = b + a - 0.5
+        x² = x*x
+        s₃ = 1.0 + (x + x²)
+        s₅ = 1.0 + (x + x²*s₃)
+        s₇ = 1.0 + (x + x²*s₅)
+        s₉ = 1.0 + (x + x²*s₇)
+        s₁₁ = 1.0 + (x + x²*s₉)
+    end
+
+    # SET W = del(b) - del(a+b)
+    t = (1.0/b)^2.0
+    w = @horner(t, .833333333333333E-01, -.277777777760991E-02*s₃, .793650666825390E-03*s₅, -.595202931351870E-03*s₇, .837308034031215E-03*s₉, -.165322962780713E-02*s₁₁)
+    w *= c/b
+
+    #COMBINING 
+    u = d*log1p(a/b)
+    v = a*(log(b) - 1.0)
+    if u <= v
```

Comment:
write as a single-line ternary
```suggestion
    return u <= v ? w - u - v : w - v - u
```